<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipWidget</class>
 <widget class="QWidget" name="ShipWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отгрузки проволоки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_Date"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="TableView" name="tableView_Ship">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="TableView" name="tableView_ShipData"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>olap/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
